app/controllers/application_controller.rb

```diff
@@ -2,4 +2,11 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  
+  def confirm_admin
+    unless current_user.admin?
+      flash[:alert] = "Test not Admin" 
```

Comment:
We may want to change the message.  What do you think about: 

> User doesn't have required permissions

Or something like that?